api/tests/organizational-entities/acceptance/application/certification-center/certification-center.admin.route.test.js

```diff
@@ -424,4 +424,114 @@ describe('Acceptance | Organization Entities | Admin | Route | Certification Cen
       expect(cancelledInvitation.updatedAt).to.deep.equal(archivedCenter.archivedAt);
     });
   });
+
+  describe('POST /api/admin/certification-centers/batch-archive', function () {
+    context('success case', function () {
+      it('returns a 200 http request', async function () {
```

Comment:
```suggestion
      it('returns a 200 HTTP status code', async function () {
```

NB : Le `200` à mettre à jour en `204` le cas échéant.